src/Octoshift/Services/AdoApi.cs

```diff
@@ -214,14 +214,22 @@ public virtual async Task<string> GetGithubHandle(string org, string teamProject
         var response = await _client.PostAsync(url, payload);
         var data = JObject.Parse(response);
 
+        // Check for error message in the response
+        var dataProvider = data["dataProviders"]["ms.vss-work-web.github-user-data-provider"];
+        var errorMessage = (string)dataProvider?["errorMessage"];
+        if (!string.IsNullOrEmpty(errorMessage))
```

Comment:
instead of using `!string.IsNullOrEmpty` use our custom extension method `errorMessage.HasValue()`. Do this in all the places where applicable for the changes in this PR.